storage/benchmarks/go.mod

```diff
@@ -7,7 +7,7 @@ replace github.com/ipld/go-ipld-prime => ../..
 replace github.com/ipld/go-ipld-prime/storage/dsadapter => ../dsadapter
 
 require (
-	github.com/ipfs/go-ds-flatfs v0.4.5
-	github.com/ipld/go-ipld-prime v0.12.3
-	github.com/ipld/go-ipld-prime/storage/dsadapter v0.0.0-20211022093231-ebf675a9bd6d
```

Comment:
@warpfork wasn't sure if/when I was supposed to do this, but the version of flatfs needed is tied to the dsadapter version. This references a commit in the PRs own history so probably better to do a merge commit here